crates/types/src/affinity.rs

```diff
@@ -11,6 +11,12 @@ pub enum Affinity {
     Sticky(DedupChainId),
 }
 
+impl From<(u32, u32)> for Affinity {
+    fn from(internal_id: (u32, u32)) -> Self {
+        Affinity::Sticky(internal_id.into())
+    }
+}
```

Comment:
if we now have a `DedupId` type, why do we need this? feels like we should prefer to convert tuple -> DedupId -> Affinity

Reply:
we don't, added this while refactoring and forgot to remove it after